endo_api/settings_prod.py

```
@@ -1,72 +1,134 @@
+        return "django.db.backends.sqlite3"
+    # Default to Postgres
+    return "django.db.backends.postgresql"
+
+
+def _db_options_from_env() -> dict:
+    # JSON override takes precedence
+    options_raw = os.environ.get("DJANGO_DB_OPTIONS")
+    if options_raw:
+        try:
+            return json.loads(options_raw)
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)